depmapomics/mutations.py

```diff
@@ -192,79 +292,34 @@ def postProcess(
-    mutations = pd.read_csv(
-        refwm.get_sample_sets().loc[sampleset, "filtered_CGA_MAF_aggregated"], sep="\t"
-    )
-    mutations = mutations.rename(columns=rename_cols).drop(
-        columns=["Center", "Tumor_Seq_Allele1"]
+    mutations = aggregateMAFs(
+        wm, sampleset=sampleset, mafcol=mafcol, keep_cols=MUTCOL_DEPMAP,
     )
 
-    mutations[mutCol] = [
-        str(i[0]) + ":" + str(i[1])
-        for i in np.nan_to_num(
-            mutations[["t_alt_count", "t_ref_count"]].values, 0
-        ).astype(int)
-    ]
-    mutations = mut.filterCoverage(mutations, loc=[mutCol], sep=":", cov=2)
-    mutations = mut.filterAllelicFraction(mutations, loc=[mutCol], sep=":", frac=0.1)
-    mutations = addAnnotation(mutations, NCBI_Build="37", Strand="+")
+    print("annotating likely immortalized status")
     mutations = annotateLikelyImmortalized(
-        mutations,
-        TCGAlocs=["TCGAhsCnt", "COSMIChsCnt"],
-        max_recurrence=0.05,
-        min_tcga_true_cancer=5,
+        mutations, hotspotcol="cosmic_hotspot", max_recurrence=0.05,
     )
 
+    print("saving somatic mutations (all)")
     mutations.to_csv(save_output + "somatic_mutations_all.csv", index=None)
     print("done")
 
-    sampleids = refwm.get_sample_sets().loc[sampleset, "samples"]
     svs = aggregateSV(
-        refworkspace,
-        sampleids,
-        all_sv_colname=sv_col,
+        wm,
+        sampleset=sampleset,
+        sv_colname=sv_col,
         save_output=save_output,
         save_filename=sv_filename,
+        sv_renaming=sv_renaming,
     )
-    return mutations, svs
-
+    print("saving svs (all)")
+    svs.to_csv(save_output + "svs_all.csv", index=None)
 
-def aggregateSV(
-    refworkspace,
-    sampleids,
-    all_sv_colname=SV_COLNAME,
-    sv_renaming=SV_COLRENAME,
-    save_output="",
-    save_filename="",
-):
-    """aggregate SVs pulled from a terra workspace
-
-    Args:
-        refworkspace (str): terra workspace where the ref data is stored
-        sampleids (list[str]): list of sequencing IDs
-        all_sv_colname (str): name of column in terra workspace that contains sv output files. Defaults to "filtered_annotated_sv"
-        save_output (str, optional): whether to save our data. Defaults to "".
-
-    Returns:
-        
-    """
-    print("aggregating SVs")
-    wm = dm.WorkspaceManager(refworkspace).disable_hound()
-    sample_table = wm.get_samples()
-    sample_table = sample_table[sample_table.index.isin(sampleids)]
-    all_svs = []
-    for name, row in sample_table.iterrows():
-        sv = pd.read_csv(row[all_sv_colname], sep="\t")
-        sv[SAMPLEID] = name
-        all_svs.append(sv)
-    all_svs = pd.concat(all_svs)
-    all_svs = all_svs.rename(columns=sv_renaming)
-    print("saving aggregated SVs")
-    all_svs.to_csv(save_output + save_filename, sep="\t", index=False)
-    return all_svs
+    return mutations, svs
 
 
 def mapBed(file, vcfdir, bed_location):
```

Comment:
it reads the guides bed for everysample??

Reply:
ahh good catch now it passes in the df